extensions/cornerstone/src/utils/stackSync/toggleStackImageSync.ts

```diff
@@ -1,90 +1,81 @@
-import calculateViewportRegistrations from './calculateViewportRegistrations';
-
-// [ {
-//   synchronizerId: string,
-//   viewports: [ { viewportId: string, renderingEngineId: string, index: number } , ...]
-// ]}
-let STACK_IMAGE_SYNC_GROUPS_INFO = [];
-
-export default function toggleStackImageSync({ toggledState, servicesManager, getEnabledElement }) {
+const disableSync = (syncName, servicesManager) => {
   const { syncGroupService, viewportGridService, displaySetService, cornerstoneViewportService } =
```

Comment:
can we move this helpers to the bottom so that the first thing we see opening this file is the actual start of the logic?

Reply:
Although I generally like that logic, it doesn't really work because you are supposed to declare functions before using them.  It isn't strictly necessary, but is preferred.

Reply:
use functions instead of const so that it get hoisted https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions#function_hoisting